index.js

```diff
@@ -0,0 +1,31 @@
+const langSec = document.querySelector(".language-section");
+const buttons = document.querySelectorAll(".button");
+const textTitle = document.querySelector(".title");
+const textDesc = document.querySelector(".description");
+
+buttons.forEach(button => {
```

Comment:
If you are using arrow functions ALWAYS wrap the arguments in parenthesis.
```suggestion
buttonEls.forEach((buttonEl) => {
```